manim/_config/utils.py

```diff
@@ -1084,8 +1084,7 @@ def quality(self):
         for qual in constants.QUALITIES:
             if all([q[k] == constants.QUALITIES[qual][k] for k in keys]):
                 return qual
-        else:
-            return None
+        return None
```

Comment:
Wouldn't the correct way to "fix this" be to add a `break` statement above? Though for some reason this did not break Manim despite being here for a long time. I would prefer this "fix" to be taken out of this PR, and be fixed (if it is actually broken) in another PR.

Reply:
As I've asked in discord, this is not broken, just written with bad practice. I think the original coder added the else when they saw the return in the loop.

Reply:
I think it looks fine to me as the else shouldn't have been there, and I think returning None is good as it is more explicit